package.json

```
@@ -16,7 +16,7 @@
     "url": "git://github.com/mattermost/desktop.git"
   },
   "config": {
-    "target": "11.4.4",
+    "target": "12.0.6",
```

```suggestion
    "target": "12.0.7",
```
shouldn't this match the electron version?